horse/models/user.py

```diff
@@ -0,0 +1,20 @@
+class User:
+    def __init__(
+            self, user_id, name, followed_users=None, liked_movies=None
+    ):
+        self.user_id = user_id
+        self.name = name
+        self._followed_users = followed_users or []
+        self._liked_movies = liked_movies or []
+
+    def get_followed_users(self):
```

Comment:
I'd prefer plain attributes or properties but I guess it doesn't matter much at this moment